theme/themes/pxt/views/card.overrides

```diff
@@ -152,6 +152,17 @@ a.ui.card:focus,
 .ui.card.link.newprojectcard {
     background: fade(@primaryColor, 90%) !important;
 }
+
+.ui.card.link.cloudprojectscard {
+    background: @cloudProjectCardBackground!important;
+}
+
+@media only screen and (min-height: tallEditorBreakpoint), (min-width: @computerBreakpoint) {
+    .ui.card.link.cloudprojectscard .icon {
+        margin-left: -1rem
```

Comment:
for some reason the new cloud svg floats right and looks silly when large. This bumps it to the left until it hits the smaller icon size, where it looks properly centered

without the extra rule:
![image](https://user-images.githubusercontent.com/18712271/125887478-d5b24191-19ad-4eb3-a1fd-6e4a2d7f23fd.png)

Reply:
with:
![image](https://user-images.githubusercontent.com/18712271/125887495-0ea599c0-5393-41e0-b769-a21bd639b1e6.png)

Reply:
How does the card look on mobile? Wondering if "Cloud Projects" wraps, or doesn't but should...

Reply:
it doesn't, I'll add it

![image](https://user-images.githubusercontent.com/18712271/125982426-927442fa-76cc-47d5-b2a5-7f2b1ba3b42a.png)